app/views/dashboard/_stripe_account_setup_banner.njk

```diff
@@ -1,12 +1,35 @@
 {% if permissions.stripe_account_details_update %}
   {% set stripeAccountSetup = account.accountSetupDetails %}
+
+  {% set panelModifierClass %}
+    {% if stripeAccountDetails.payouts_enabled === false %}
```

Comment:
As the message provided is:
> To start taking payments again, please add:

I think this should be checking for `payments_enabled` rather than `payouts_enabled`, if this was lifted and assigned to a constant at the top it would only have to change once! 

(Stripe accounts can continue to process payments until the `payments_enabled` flag is false.

Reply:
As discussed, using `charges_enabled` flag.